### ISSUE TITLE
Crash when opening for the first time with Mac

### ISSUE THREAD

**Issue by BFlavien** [PixelVision8/PixelVision8#447]
**Describe the bug**
Before running for the first time, it crashes: `x.x.x quit unexpectedly.`

I am using a MacBook Pro, running MacOS Big Sur (11.1). Details about my configuration can be found in the full report at the end of the issue.

**To reproduce**
1. Download v1.0.0 release from https://github.com/PixelVision8/PixelVision8/releases/tag/v1.0.0
2. Rename to add `.zip` at the end of the file name
3. Unzip
4. Double-click on `Pixel Vision 8` (still in the Downloads folder, but I have the same behaviour if I move it to Application folder)
5. _"Pixel Vision 8" cannot be opened because the developer is not verified._ 
![image](https://user-images.githubusercontent.com/4741522/104955171-0e1d5000-59ca-11eb-8d52-67c3ce8916fe.png)
6. Click **Cancel**
7. Go to **Security & Privacy**
![image](https://user-images.githubusercontent.com/4741522/104955329-5d638080-59ca-11eb-9d4c-432fb67619f1.png)
8. Click **Open Anyway**
9. Displays warning about developer not being verified
![image](https://user-images.githubusercontent.com/4741522/104955392-7e2bd600-59ca-11eb-85c8-a7698dbe06d6.png)
10. Click **Open**
11. Crash 💥  (full report at the end)
![image](https://user-images.githubusercontent.com/4741522/104955447-913ea600-59ca-11eb-97f5-013eda07b1fb.png)

**Expected behavior**
I expect the app to run.

**Additional notes**

I also tried with latest version (1.0.6)

Only step 11 is different (full report at the end):
![image](https://user-images.githubusercontent.com/4741522/104956090-b54eb700-59cb-11eb-863d-b9fe3f63ba38.png)

[Full report for V1.0.0](https://gist.github.com/BFlavien/2a121149bce672da54878b7d406db31d)
[Full report for V1.0.6](https://gist.github.com/BFlavien/8d2a94656100dbfdd11a768278610bd2)

PS: Thanks for your work man ! It looks awesome !

**Comment by BFlavien** [comment]
I just tried with V1.0.7, and still have the same issue :)

**Comment by BFlavien** [comment]
I just tried with V1.0.9, and still have the same issue :)

**Comment by jessefreeman** [comment]
Does it work after the first launch?

**Comment by BFlavien** [comment]
Nope, I always repeated the same steps I listed in the first message, and always get the same result.

**Comment by decasso** [comment]
Hi, Pixel Vision 8 crash on start in my iMac with Mac OS 10.15.7. I tried 1.0.0 and 1.0.12
I checked security and I gave permission to execute. It always crash if I run Pixel Vision 8.app, but I used Finder to open content file of Pixel Vision 8, I gave double click to /Content/Mac OS/Content/Pixel Vision 8, I suppose it's an binary file. In this way, Pixel Vision 8 always ran but with an advice of warning.  I don't know where is my files in Workspace. Pixel Vision 8.app was in Documents, not in Applications, to quick search in Finder. For me, Pixel Vision 8 only run if I execute content file in .app, before and after Pixel Vision.app always crash in execute. Sorry for my bad english. 
<img width="972" alt="Captura de Pantalla 2021-04-11 a la(s) 18 27 44" src="https://user-images.githubusercontent.com/34075550/114326867-fbf80d00-9af3-11eb-8b6a-7a9ac2843746.png">
<img width="967" alt="Captura de Pantalla 2021-04-11 a la(s) 18 28 00" src="https://user-images.githubusercontent.com/34075550/114326873-fe5a6700-9af3-11eb-940f-9082e58e366c.png">
<img width="961" alt="Captura de Pantalla 2021-04-11 a la(s) 18 28 49" src="https://user-images.githubusercontent.com/34075550/114326880-06b2a200-9af4-11eb-8d5e-8ca6d5439ef7.png">
<img width="964" alt="Captura de Pantalla 2021-04-11 a la(s) 18 29 16" src="https://user-images.githubusercontent.com/34075550/114326888-0c0fec80-9af4-11eb-91de-3869d95e9b76.png">
<img width="2045" alt="Captura de Pantalla 2021-04-11 a la(s) 18 30 01" src="https://user-images.githubusercontent.com/34075550/114326892-0f0add00-9af4-11eb-9350-8d67cd66af84.png">

**Comment by raphaklaus** [comment]
I can confirm that @decasso's workaround works.

**Comment by DannyCork** [comment]
same issue, crash upon first execution
Pixel Vision 8 v1.0.18
macOS Monterey Version 12.1
Intel

`-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Pixel Vision 8 [1894]
Path:                  /private/var/folders/*/Pixel Vision 8.app/Contents/MacOS/Pixel Vision 8
Identifier:            com.pixelvision8
Version:               1.0 (1)
Code Type:             X86-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2021-12-14 00:42:44.9053 +0000
OS Version:            macOS 12.1 (21C52)
Report Version:        12
Bridge OS Version:     6.1 (19P647)
Anonymous UUID:        B59B6F58-08E8-559D-83B4-AC82A29B644D


Time Awake Since Boot: 1500 seconds

System Integrity Protection: disabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Application Specific Information:
abort() called


Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib        	    0x7ff819df2112 __pthread_kill + 10
1   libsystem_pthread.dylib       	    0x7ff819e28214 pthread_kill + 263
2   libsystem_c.dylib             	    0x7ff819d74d10 abort + 123
3   libcoreclr.dylib              	       0x1015d28ae PROCAbort + 30
4   libcoreclr.dylib              	       0x1011a55f7 PROCEndProcess(void*, unsigned int, int) + 215
5   libcoreclr.dylib              	       0x1015d535c UnwindManagedExceptionPass1(PAL_SEHException&, _CONTEXT*) (.cold.1) + 44
6   libcoreclr.dylib              	       0x1013f4946 UnwindManagedExceptionPass1(PAL_SEHException&, _CONTEXT*) + 822
7   libcoreclr.dylib              	       0x1013f49d6 DispatchManagedException(PAL_SEHException&, bool) + 134
8   libcoreclr.dylib              	       0x101358244 IL_Throw(Object*) + 548
9   ???                           	       0x108693c71 ???
10  ???                           	       0x10868945f ???
11  ???                           	       0x108774e99 ???
12  ???                           	       0x10876bdde ???
13  ???                           	       0x108000408 ???
14  ???                           	       0x107ffff86 ???
15  ???                           	       0x107fe696a ???
16  libcoreclr.dylib              	       0x101495369 CallDescrWorkerInternal + 124
17  libcoreclr.dylib              	       0x1012e9e5f MethodDescCallSite::CallTargetWorker(unsigned long const*, unsigned long*, int) + 1519
18  libcoreclr.dylib              	       0x1012e9e5f MethodDescCallSite::CallTargetWorker(unsigned long const*, unsigned long*, int) + 1519
19  libcoreclr.dylib              	       0x1012e9e5f MethodDescCallSite::CallTargetWorker(unsigned long const*, unsigned long*, int) + 1519
20  libcoreclr.dylib              	       0x1012e9e5f MethodDescCallSite::CallTargetWorker(unsigned long const*, unsigned long*, int) + 1519
21  libcoreclr.dylib              	       0x1012e9e5f MethodDescCallSite::CallTargetWorker(unsigned long const*, unsigned long*, int) + 1519
22  libcoreclr.dylib              	       0x1012e9e5f MethodDescCallSite::CallTargetWorker(unsigned long const*, unsigned long*, int) + 1519
23  libcoreclr.dylib              	       0x1012e9e5f MethodDescCallSite::CallTargetWorker(unsigned long const*, unsigned long*, int) + `
